hc/front/tests/test_add_channel.py

```diff
@@ -43,13 +43,18 @@ def test_bad_kinds_not_working(self):
         url = "/integrations/add_%s/" % bad_kind
         r = self.client.get(url)
         self.assertEqual(r.status_code, 404)
-        
-
-    
-
-
+         
+    ### Test that the team access works
+    def test_can_add_channel_using_team_access(self):
+        url = "/integrations/add/"
+        form = {"kind": "email", "value": "bob@example.org"}
 
+        # Bob is in alice's team so he should be able to add a channel
+        self.client.login(username="bob@example.org", password="password")
+        r = self.client.post(url, form)
```

Comment:
hey @collinmutembei Could that be addressed by the assert I made below, by confirming that the integration belongs to alice?

Reply:
@briankabiro Does a channel have a relation with an integration?

Reply:
🤔 Come to think of it; it makes sense @briankabiro